packages/status-js/src/client/chat.ts

```diff
@@ -208,7 +208,12 @@ export class Chat {
     }
 
     await this.client.waku.store.queryWithOrderedCallback(
-      [createDecoder(this.contentTopic, this.symmetricKey)],
+      [
+        createDecoder(this.contentTopic, this.symmetricKey, {
+          clusterId: 16,
+          shard: 32,
```

Comment:
You're repeating this settings in different files. Maybe it's worth to centralize it in a file. Change it once, affects everywhere.

Reply:
Just like the encoder (`createDecoder()`) itself – this wasn't a concern of mine for this PR.